python/src/trezorlib/cli/ethereum.py

```diff
@@ -164,14 +167,96 @@ def _format_access_list(
     ]
 
 
+def _hex_or_file(data: str) -> bytes:
+    try:
+        path = Path(data)
+        if path.is_file():
+            return path.read_bytes()
+    except Exception:
```

Comment:
Is there even any `Exception` that can be triggered? `Path()` can take any `str` and it will not raise

Reply:
are you sure it won't raise if, e.g., there are forbidden filename characters?
if yes, then you are right and i'll change this as you suggest

Reply:
looks good.
changed code in rebase